src/python/grpcio/tests/unit/framework/interfaces/base/test_cases.py

```diff
@@ -54,13 +54,13 @@ def serialize_request(self, request):
     return request + request
 
   def deserialize_request(self, serialized_request):
-    return serialized_request[:len(serialized_request) / 2]
+    return serialized_request[:int(len(serialized_request) / 2)]
```

Comment:
We *could* `from __future__ import division` and change this to a `... // 2` instead of adding the integer cast. But then that sets us up to want to say, "We're going to do this everywhere."

@nathanielmanistaatgoogle Got a preference?